.github/scripts/Tests/cost-walker-slug.Tests.ps1

```
@@ -0,0 +1,231 @@
+        if (-not (Test-Path -LiteralPath $script:LibPath)) {
+            throw "cost-walker library not found at $script:LibPath"
+        }
+        . $script:LibPath
+
+        # Amendment A2 comparison rule: a derived slug's tail is compared
+        # exactly, its drive letter case-insensitively. The drive character is
+        # the one position whose case a caller-supplied path can legitimately
+        # differ on (an MSYS path carries a lowercase drive; the session that
+        # ran there recorded an uppercase one), and the on-disk lookup is
+        # case-insensitive there. Everywhere else, case is load-bearing.
+        function script:Should-MatchSlug {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Should-MatchSlug' uses an unapproved verb.

(PSUseApprovedVerbs)